### PR TITLE
Mode cleanup

### PR DESCRIPTION
**Pull request by knausj85**
- Switch gdb and windbg to tags

Addresses the remainder of https://github.com/knausj85/knausj_talon/issues/603

Probably most relevant to @fidgetingbits

### CODE REVIEW

**Comment by fidgetingbits** [comment]
Looks good to me, though didn't test it

File: core/modes/modes.py [hunk 1]
```diff
@@ -8,7 +8,6 @@
     "gdb": "a way to force gdb commands to be loaded",
```
Comment (splondike) [inline core/modes/modes.py:8]: I think we want to delete this gdb line as well.

**Review by splondike — APPROVED** [review]
Gave it a bit of a test (just checked that when the tag was enabled the right commands ran), looks good! Probably just want to remove that leftover gdb mode.

File: core/modes/modes.py (hunk 1 above)
Comment (knausj85) [reply core/modes/modes.py]: Fixed, thanks

File: core/modes/modes.py [hunk 2]
```diff
@@ -5,10 +5,8 @@
 modes = {
     "admin": "enable extra administration commands terminal (docker, etc)",
     "debug": "a way to force debugger commands to be loaded",
-    "gdb": "a way to force gdb commands to be loaded",
```
Comment (auscompgeek) [inline core/modes/modes.py:8]: I think we forgot to remove the voice commands activating this mode

https://github.com/knausj85/knausj_talon/blob/99869cbaebe68ec43f4ed992e0317deb9bd01070/core/modes/language_modes.talon#L3-L4

File: apps/gdb/gdb.py [hunk 3]
```diff
@@ -1,13 +1,30 @@
-from talon import Context, actions
+from talon import Context, Module, actions
 
-ctx = Context()
+mod = Module()
+mod.tag("gdb", "Tag to enabled gdb-related functionality")
```
Comment (auscompgeek) [inline apps/gdb/gdb.py:4]: Looking around I think this conflicts with the `gdb` language tag 😬

**Pull request merged**